src/hw/connectManager.js

```diff
@@ -28,32 +28,37 @@ const cmd = ({ devicePath }: Input): Observable<ConnectManagerEvent> =>
         .pipe(delay(1000))
         .subscribe((e) => o.next(e));
 
-      const sub = from(getDeviceInfo(transport))
+      const sub = defer(() => from(getAppAndVersion(transport)))
         .pipe(
-          concatMap((deviceInfo) => {
+          concatMap((appAndVersion) => {
```

Comment:
We need to rework this to improve that inside getDeviceInfo as this will break older LNS versions
Once we do this, this file should have no diff